src/debug.c

```diff
@@ -62,6 +62,7 @@
 #include "constants/species.h"
 #include "constants/weather.h"
 #include "save.h"
+#include "tv.h"
```

Comment:
Was this include needed?

Reply:
Ah ok :)

Reply:
... but I didn't actually need to count the digits, because `STR_CONV_MODE_LEFT_ALIGN` doesn't pad to the right :facepalm: